api-compose-object.go

```diff
@@ -173,6 +173,7 @@ func (d *DestinationInfo) getUserMetaHeadersMap(withCopyDirectiveHeader bool) ma
 // server-side copying APIs.
 type SourceInfo struct {
```

Comment:
We should deprecate this SourceInfo instead it should become SourceOptions and DestinationInfo should become DestinationOptions

Reply:
@vadmeste can you make this change? ^^

Reply:
> @vadmeste can you make this change? ^^

It is already done in a separate PR,  https://github.com/minio/minio-go/pull/1305.. it just needs API change approval